unstructured/ingest/runner/gcs.py

```diff
@@ -1,15 +1,21 @@
 import logging
 import typing as t
+from dataclasses import dataclass
 
 from unstructured.ingest.logger import ingest_log_streaming_init, logger
 from unstructured.ingest.runner.base_runner import FsspecBaseRunner
 from unstructured.ingest.runner.utils import update_download_dir_remote_url
 
+if t.TYPE_CHECKING:
+    from unstructured.ingest.connector.gcs import SimpleGcsConfig
 
+
+@dataclass
 class GCSRunner(FsspecBaseRunner):
+    fsspec_config: t.Optional["SimpleGcsConfig"] = None
```

Comment:
Given line 10,
```suggestion
    fsspec_config: t.Optional[SimpleGcsConfig] = None
```

Reply:
When you use 
```
if t.TYPE_CHECKING:
    from unstructured.ingest.connector.gcs import SimpleGcsConfig
```
The dependency when used in the file needs to be in quotes. Otherwise it tried to import it.

Reply:
For reference: https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING

Reply:
Checked it now and sounds good 👍